index/index.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+// Need to change so it gets every class and add better variable names
+// also make a little more efficient and cleaner 
+
+const AWS = require('aws-sdk');
+AWS.config.update({ region: "us-east-2" });
+let lambda = new AWS.Lambda();
```

Comment:
For improved performance, I highly recommend creating and importing a separate cache module for the AWS imports. Check out [the cache module in Banner-Proxy](https://github.com/schedulemaker/banner-proxy/blob/master/index/cache.js) for and example of how to do it in NodeJS. And then you would import it in index.js the same way:
```
if (typeof(cache) === 'undefined'){
   var cache = require('./cache');
}
```
This way the next time the Lambda container is used (almost always), you won't waste time importing the SDK and creating the Lambda object.